### ISSUE TITLE
prune parmeters 

### ISSUE THREAD

**Issue by byhashdong** [allada/bsc-archive-snapshot#20]
Could you share your prune parmeters ? 
When i run erigon based the snapshot, it indicates me wrong prune parmeters.

**Comment by allada** [comment]
I use the default parameters in the [docker-compose.yml](https://github.com/ledgerwatch/erigon/blob/devel/docker-compose.yml) file. The only thing I modify in the script is it adds: `--chain bsc` to the `erigon` command.

**Comment by allada** [comment]
I think I see the errors you are receiving. I have fixed it in a patch I just pushed https://github.com/allada/bsc-archive-snapshot/commit/188a3feb4009bd6e7cf12c769b08b8df569fdbed

erigon made some changes and it appears the default of this flag was changed, and was notifying users that there's a problem from the database version they are using and the launched process flag.

It should be fixed now, if you apply that patch then run:
```sh
docker-compose stop
ERIGON_FLAGS=--syncmode=full XDG_DATA_HOME=/erigon/data docker-compose up -d
```
as root in the /erigon/erigon directory, it should work.
